otter/convergence/service.py

```diff
@@ -155,6 +156,9 @@ def execute_convergence(tenant_id, group_id, build_timeout,
                                            status=ScalingGroupStatus.ACTIVE))
             yield cf_msg('group-status-active',
                          status=ScalingGroupStatus.ACTIVE.name)
+        else:
+            # Clear servers cache that removes deleted servers
+            yield cache_class(tenant_id, group_id).delete_servers()
```

Comment:
Apologies - I thought `delete_servers` deleted all servers from the cache?  If not, then I guess please ignore my comment above about deleted servers.  (And also, if that's the case, should we also have a delete all servers function that gets called when the group gets deleted for real?)